test/lib/next-test-utils.ts

```diff
@@ -1553,8 +1553,9 @@ export async function assertNoConsoleErrors(browser: BrowserInterface) {
       log.source === 'warning' ||
       (log.source === 'error' &&
         // These are expected when we visit 404 pages.
-        log.message !==
-          'Failed to load resource: the server responded with a status of 404 (Not Found)')
+        log.message.startsWith(
+          'Failed to load resource: the server responded with a status of 404'
+        ))
```

Comment:
Message format is slightly different on deploys: https://github.com/vercel/next.js/actions/runs/12712892279/job/35440856951?pr=74748#step:29:105